Assets/Scripts/GameData/Creatures/GenericStats.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SwordAndBored.GameData.Creatures
+{
+    class GenericStats : IStats
+    {
```

Comment:
Can you explain why floats vs ints are used for different stats?

Reply:
Since Accuracy and Evasion affect your chance to hit and are a percent then the others are just numbers